### DISCUSSION TITLE
Inquiry on PR Merge Release Timeline

### DISCUSSION THREAD

**Discussion by BestWeb3** [rainbow-me/rainbowkit#2243] [Q&A] [answered]
Hey @DanielSinclair ,

I hope you are doing well. I am writing to inquire about the expected release version for the recent updates included in our https://github.com/rainbow-me/rainbowkit/pull/2233. Could you please let us know in which version we can expect these changes to be available, and if possible, share the anticipated timeline for the release of that version?

Your clarification would greatly help us in planning our next steps accordingly.

Thank you for your assistance.

Best Regards,
BestWallet Team

**Comment by BestWeb3** [comment]
Hey @DanielSinclair ,

I hope this message finds you well. I wanted to follow up on our earlier inquiry regarding the release version for the updates included in #2233.

Could you please confirm which version will include these changes and, if possible, provide an updated timeline for its release? This information would be greatly appreciated as it will help us better plan our next steps.

Thank you again for your support, and please let us know if you need any additional information from our end.

Looking forward to your response.

Best regards,
BestWallet Team

**Comment by DanielSinclair** [comment, marked as answer]
Released as [2.2.1](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%402.2.1)
